source/includes/administration/_environments.md

```diff
@@ -116,18 +130,22 @@ Attributes | &nbsp;
 `id`<br/>*UUID* | The id of the environment
 `name`<br/>*string* | The name of the environment
 `description`<br/>*string* | The description of the environment
+`metadata`<br/>*object* | Additional information about the environment
 `membership`<br/>*string* | Type of membership of the environment. (e.g. ALL_ORG_USERS, MANY_USERS)
 `creationDate`<br/>*string* | The date in [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) that the environment was created
 `organization`<br/>*[Organization](#administration-organizations)* | The organization of the environment<br/>*includes*: `id`, `name`, `entryPoint`
-`serviceConnection`<br/>*[ServiceConnection](#administration-service-connections)* | The service connection of the environment<br/>*includes*: `id`, `name`
-`users`<br/>*Array[[User](#administration-users)]* | The users that are members of the environment<br/>*includes*: `id`, `username`
-`roles`<br/>*Array[[Role](#administration-roles)]* | The roles of the environment with all the users assigned to them.<br/>*includes*: `id`, `name`, `isDefault`, `users.id`, `users.name`
+`allowExternalMembers`<br/>*boolean* | Indicates if the environment supports external members or not
+`state`<br/>*string* | Indicates the state of the environment. Possible states are PROVISIONING, PROVISIONED, ERROR
+`serviceConnection`<br/>*[ServiceConnection](#administration-service-connections)* | The service connection of the environment<br/>*includes*: `id`, `name`, `serviceCode`, `type`
 
 <!-------------------- CREATE ENVIRONMENT -------------------->
 
 
 ### Create environment
 
+Environments are created asynchronously to the underlying service. When creating an environment any underlying actions performed by the plugin is done asynchronously to the creation of the model, and its progress is reflected in the state of the environment. 
+
+
```

Comment:
will there be more for create environment?

Reply:
I dont believe the request requirements have changed